### PR TITLE
Fix comment on the 'concatAndUglify' function

### PR DESCRIPTION
**Pull request by josepot**

### CODE REVIEW

File: src/rangle-gulp.js [hunk 1]
```diff
@@ -190,7 +190,8 @@ exports.beautify = function (options) {
   };
 };
 
-// Makes a task that runs JSBeautify on all script and test files.
+// Makes a task that concats all the non 'test' js files, and publishes both:
+// the concat and the 'concated' and 'uglified' version (.min.js)
```
Comment (SethDavenport) [inline src/rangle-gulp.js]: This wording is a bit odd.  How about:

// Makes a task that concatenates all the non 'test' js files, and publishes both:
// a version which is concatenated and uglified, and a version which is just concatenated.

**Comment by yuri** [comment]
@josepot The phrasing _is_ a little odd. Can you fix it?

**Comment by josepot** [comment]
@SethDavenport sorry it took me so long to address this, I haven't seen your comment until now. I just changed it, thanks! 

@yuri it's fixed now.

**Comment by SethDavenport** [comment]
:+1:

**Pull request merged**